src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -77,7 +77,12 @@ internal bool Initialize(EngineController controller, WorkspaceModel workspace)
                 {
                     var node = workspace.Nodes.FirstOrDefault(n => n.GUID.Equals(nodeGuid));
                     if (node != null)
+                    {
                         node.ClearDirtyFlag();
+
+                        // Set IsEvaluating to true, so that node's preview bubble won't crash.
+                        node.IsEvaluating = true;
```

Comment:
Why we need this new flag? We could make `CachedValue` be invalid once node is modified.